pkgs/tools/audio/tts/default.nix

```diff
@@ -11,25 +11,20 @@
 #
 # If you upgrade from an old version you may have to delete old models from ~/.local/share/tts
 # Also note that your tts version might not support all available models so check:
-#   https://github.com/coqui-ai/TTS/releases/tag/v0.1.2
+#   https://github.com/coqui-ai/TTS/releases/tag/v0.1.3
 #
 # For now, for deployment check the systemd unit in the pull request:
 #   https://github.com/NixOS/nixpkgs/pull/103851#issue-521121136
 
-let
-  python3 = python38;
-in python3.pkgs.buildPythonApplication rec {
+python3.pkgs.buildPythonApplication rec {
   pname = "tts";
-  version = "0.1.2";
-
-  # https://github.com/coqui-ai/TTS/issues/570
-  disabled = python3.pythonAtLeast "3.9";
+  version = "0.1.3";
 
   src = fetchFromGitHub {
     owner = "coqui-ai";
     repo = "TTS";
     rev = "v${version}";
-    sha256 = "1qgiaqn7iqxyf54qgnpmli69nw9s3gmi9qv874jsgycykc10hjg4";
+    sha256 = "sha256-fl4nrBCznalzc8wu+30rgORID9TW7tvJikqik7TYjp4=";
```

Comment:
nix-shell:~/.cache/nixpkgs-review/pr-131598]$ nix-shell -p tts
these 2 derivations will be built:
  /nix/store/x1vbjrii5cj53bb6qyrs62v9ba521rdm-source.drv
  /nix/store/77zlx3g5b7wifvkvy0g3vn7qaapyf0z8-tts-0.1.3.drv
warning: unknown setting 'experimental-features'
building '/nix/store/x1vbjrii5cj53bb6qyrs62v9ba521rdm-source.drv'...


trying https://github.com/coqui-ai/TTS/archive/v0.1.3.tar.gz
trying https://github.com/coqui-ai/TTS/archive/v0.1.3.tar.gz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
                                 Dload  Upload   Total   Spent    Left  Speed
100   120    0   120    0     0    446      0 --:--:-- --:--:-- --:--:--   444
100   120    0   120    0     0    446      0 --:--:-- --:--:-- --:--:--   444
100 13.9M    0 13.9M    0     0  9256k      0 --:--:--  0:00:01 --:--:-- 15.6M
100 13.9M    0 13.9M    0     0  9256k      0 --:--:--  0:00:01 --:--:-- 15.6M
unpacking source archive /build/v0.1.3.tar.gz
unpacking source archive /build/v0.1.3.tar.gz
hash mismatch in fixed-output derivation '/nix/store/0lwp3b1k7w00hk3kygf6mg1njkb85vrg-source':
  wanted: sha256:17lfv2s978jaib4xpvnnsh7lir405dyznbncfdrsk7dk22n2fpky
  got:    sha256:0akhiaaqz53bf5zyps3vgjifmgh5wvcc9r4lrq9hmj3dds03vkjq
cannot build derivation '/nix/store/77zlx3g5b7wifvkvy0g3vn7qaapyf0z8-tts-0.1.3.drv': 1 dependencies couldn't be built
error: build of '/nix/store/77zlx3g5b7wifvkvy0g3vn7qaapyf0z8-tts-0.1.3.drv' failed

Did they maybe changed the release again?

Reply:
Possibly, I'm not sure. Pushed an update.